commands/utils/packagehandlers/gopackagehandler.go

```diff
@@ -2,14 +2,21 @@ package packagehandlers
 
 import (
 	"github.com/jfrog/frogbot/commands/utils"
-	"strings"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const (
+	GoPackage = "github.com/golang/go"
 )
 
 type GoPackageHandler struct {
 	GenericPackageHandler
 }
 
 func (golang *GoPackageHandler) UpdateImpactedPackage(impactedPackage string, fixVersionInfo *utils.FixVersionInfo, extraArgs ...string) error {
-	impactedPackage = strings.Trim(impactedPackage, "v")
+	if impactedPackage == GoPackage {
```

Comment:
where did `impactedPackage = strings.Trim(impactedPackage, "v")` disappear?

Reply:
The impacted package is the vulnerable package name, we should not trim its name. Also, the fix version is coming from Xray without the "v" prefix of Go, so it's unnecessary.